examples/getting-started-movielens/03-Training-with-PyTorch.ipynb

```diff
@@ -37,45 +37,36 @@
     "\n",
```

Comment:
<p>Could we leverage the schema file for the preprocess batch function? Do we Tag the columns in the NVTabular notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>turns out that once I properly tag the data I don't even have to do the map thing! 🥳 the <code>Merlin Dataloader</code> automatically process the column to the 2nd position in the returned batch and removes it from inputs! </p>

Reply:
that is great!